docs/Changelog.md

```diff
@@ -10743,7 +10743,7 @@ This version of the operator has been available since version 11 of the default
   output_shape can also be explicitly specified in which case pads values are auto generated using these equations:
   
     total_padding[i] = stride[i] * (input_size[i] - 1) + output_padding[i] + ((kernel_shape[i] - 1) * dilations[i] + 1) - output_shape[i]
-    If (auto_pads != SAME_UPPER): pads[start_i] = total_padding[i]/2; pads[end_i] = total_padding[i] - (total_padding[i]/2)
+    If (auto_pads == SAME_UPPER): pads[start_i] = total_padding[i]/2; pads[end_i] = total_padding[i] - (total_padding[i]/2)
```

Comment:
Is this intentional? Why change this ?

Reply:
It is another old issue related to ConvTranspose which hasn't been resolved: https://github.com/onnx/onnx/issues/1609 There is also a PR for it: https://github.com/onnx/onnx/pull/2097
The original reason is the description is not consistent with the example. I thought it's a good chance to push this forward so I added this fix in this PR as well.